src/Phinx/Db/Table/Column.php

```diff
@@ -151,7 +151,7 @@ public function getType()
     /**
      * Sets the column limit.
      *
-     * @param integer $limit
+     * @param mixed $limit integer|string - can be string for SET and ENUM
```

Comment:
This is pretty weird to me, I don't think using `limit` for ENUM options makes sense... and shouldn't ENUM options be an array, not a string?